Dockerfile

```diff
@@ -1,19 +1,37 @@
-FROM node:16-bullseye
+FROM debian:bullseye
```

Comment:
interesting, any reason for this change?

Reply:
Well, we need Node and Postgres and pgbouncer all in the same container, and while the Node image was based on the same version of Debian as this, it ripped out things from Debian that Node does not need.

I figured going from a more "standard" base that I can look up problems online and be more sure they'll actually apply would reduce the total workload and maintenance burden longer-term.

Reply:
Why do we need Node/Postgres/pgbouncer all in the same container? Can't a `docker-compose` do the trick?

Reply:
@mtp1376 that was discussed in [this RFC](https://github.com/iasql/iasql-engine/blob/main/rfcs/005%20-%20Unsurprising%20Functions%20and%20Scalability%20RFC.md#final-proposal) where this is the first PR to begin implementing it.

Basically by tightly coupling the engine, postgres, and pgbouncer, we can eliminate the unexpected behavior of multiple statements involving IaSQL RPC functions not having the mutated state available to follow-up statements *and* we can resolve scalability issues by letting the engine use more than one core on a machine, and eventually partition users between different machines running this setup (since no one user database should consume an entire machine, but also no user database has any interaction with any other user database so they don't need to be in the same database server).